packages/pirania-app/files/usr/libexec/rpcd/pirania-app

```diff
@@ -27,6 +27,35 @@ local function shell(command)
     return result
 end
 
+local function read_governance(msg)
+    local result = {}
+    local path = '/etc/pirania/governance.json'
+    result = utils.readJsonFile(path)
```

Comment:
you don't need to read it as json, you can just print it out, as it is already a json :)

Reply:
it needs to be parsed though.

Reply:
If it is already in the filesystem, it should have been parsed before... so not sure if it is needed